test/browser.js

```diff
@@ -226,6 +227,10 @@ test.failing('FormData with searchParams ("multipart/form-data" parser)', withPa
 					resolve([null, error_]);
 				}
 			});
+			busboy.on('finish', () => {
+				response.writeHead(303, {Connection: 'close', Location: '/'});
+				response.end();
+			});
```

Comment:
Why is listening for `finish` necessary?

It's not clear to me when the `response.end()` on line 232 will be called vs the one on line 238. Is calling `response.end()` twice safe?

Reply:
I deleted `response.end()` on line 238. But we need `response.writeHead(303, {Connection: 'close', Location: '/'});` to safely end response, without that -  puppeteer is crashing on `page.close()` line in withPage helper.